src/coap_openssl.c

```diff
@@ -68,6 +68,18 @@
 #define TLSEXT_TYPE_server_certificate_type 20
 #endif
 
+#ifndef COAP_OPENSSL_CIPHERS
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L
+#define COAP_OPENSSL_CIPHERS "TLSv1.3:TLSv1.2:TLSv1.0:!NULL"
```

Comment:
I wonder if we really want anything from the deprecated TLS 1.0?

Reply:
Good question - I think drop TLSv1.0.  I will do some tests here to double check that we are not reliant on something.

Reply:
Dropping TLSv1.0 has not caused any issues during my testing.